eng/versioning/version_client.txt

```diff
@@ -23,7 +23,7 @@ com.azure:azure-identity;1.0.4;1.1.0-beta.3
 com.azure:azure-messaging-eventhubs;5.0.2;5.1.0-beta.1
 com.azure:azure-messaging-eventhubs-checkpointstore-blob;1.0.2;1.1.0-beta.1
 com.azure:azure-messaging-servicebus;1.0.0-beta.1;1.0.0-beta.1
-com.azure:azure-search-documents;11.0.0-beta.1;11.0.0-beta.2
+com.azure:azure-search-documents;11.0.0-beta.1;1.0.0-beta.2
```

Comment:
@sima-zhu I'm curious about these versions. com.azure:azure-search-documents is a renamed com.azure:azure-search and [com.azure:azure-search was released as 11.0.0-beta.1](https://search.maven.org/search?q=g:com.azure%20AND%20a:azure-search) however there is no released com.azure:azure-search-documents so the current and dependency versions should be same. (Obviously nothing in the tree currently uses the dependency version of this otherwise we would have seen compilation errors.) My recommendation here is to set both the current and dependency versions to 1.0.0-beta.1.

Reply:
The version name aligns with all languages. 
@tg-msft might have further explanation on the decision.

Reply:
This was something we discussed in the Search crew and decided to go with beta.2 to indicate to customers that it's the second preview of this library.  The hope was to make it clearer to anyone who goes digging through history that something came before this.  Sima is correct that all the other languages are aligning to Preview.2, beta.2, b2, etc.